app/models/item.rb

```diff
@@ -1,3 +1,14 @@
 class Item < ApplicationRecord
   belongs_to :supermarket
+
+  has_many :customer_items
+  has_many :customers, through: :customer_items
+
+  def supermarket_name
+    Supermarket.find(supermarket_id).name
```

Comment:
Nice job creating this method to get the name of the supermarket. It would also be acceptable to call `item.supermarket.name` in your `items/index.html.erb`.